samples/docker/build_docker.sh

```diff
@@ -1,30 +1,49 @@
 #!/bin/bash
-# Copyright 2020 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
It's my understanding that we should not change dates in licence headers. Or was this a correction of a previously incorrect year?